tests/assistant_spec.js

```diff
@@ -831,6 +868,157 @@ describe('TranslationAssistant (math-translate)', function() {
     });
 });
 
+describe('TranslationAssistant (maybe-math-translate)', function() {
+
+    it('should translate math according to a template', function() {
+        // Lang 'id' is in both MAYBE_TIMES_AS_CDOT and MAYBE_DIV_AS_COLON
```

Comment:
You could also add an assert to that effect, in case it's ever removed, it'll be more obvious why the test breaks.

`assert(MATH_RULES_LOCALES.MAYBE_TIMES_AS_CDOT.includes(lang))`?

Reply:
Good idea, I've done it for this first test.